scripts/undeploy.actions.js

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const fs = require('fs')
+const childProcess = require('child_process')
+const config = require('./script.config')
+
+function undeployActionsSync () {
+  if (!fs.existsSync(config.distWskManifestFile) || !fs.statSync(config.distWskManifestFile).isFile()) {
+    throw new Error('Cannot undeploy actions, seems like they are not deployed?')
```

Comment:
Well when deploying we are actually generating a .manifest-dist.yml which contains more things, and here I am looking for this one. I know it's not ideal but if the user redeploys it will generate this one and then he can undeploy. And if we undeploy we delete it. Wdyt, do you have a better idea?

Reply:
Even in that case the error message gives better idea that what is the problem than just saying not able to undeploy.